internal/pkg/util/fs/squashfs/squashfs.go

```diff
@@ -0,0 +1,33 @@
+package squashfs
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/runtime/engines/config"
+	singularityConfig "github.com/sylabs/singularity/pkg/runtime/engines/singularity/config"
```

Comment:
Probably better not to use caps in package names

```suggestion
	singularityconfig "github.com/sylabs/singularity/pkg/runtime/engines/singularity/config"
```

Reply:
fixed